libs/@guardian/libs/src/consent-management-platform/lib/sourcepointConfig.ts

```diff
@@ -11,3 +11,10 @@ export const ENDPOINT = isGuardianDomain()
 	? 'https://sourcepoint.theguardian.com'
 	: 'https://cdn.privacy-mgmt.com';
 export type EndPoint = typeof ENDPOINT;
+
+// https://docs.sourcepoint.com/hc/en-us/articles/4405397484307-Event-callbacks-CMP#h_01FTY32EGZ7H3SS192MEE6SNCP
+export enum SourcePointChoiceTypes {
+	AcceptAll = 11,
+	RejectAll = 13,
+	Dismiss = 15,
+}
```

Comment:
We generally want to avoid using enums due to the way they're represented once compiled can cause complications on the way we think they work.

Suggest using `as const` instead

See https://tduyng.medium.com/why-you-might-be-using-enums-in-typescript-wrong-6d9c5742db5a

```suggestion
// https://docs.sourcepoint.com/hc/en-us/articles/4405397484307-Event-callbacks-CMP#h_01FTY32EGZ7H3SS192MEE6SNCP
export const SourcePointChoices = {
  AcceptAll: 11,
  RejectAll: 13,
  Dismiss: 15
} as const;
```

Reply:
Thanks for this. Just updated the PR.